swmmio/tests/test_dataframes.py

```diff
@@ -258,3 +262,71 @@ def test_polygons(test_model_02):
     assert poly1.equals(test_model_02.inp.polygons)
 
     # print()
+
+def test_inp_sections():
```

Comment:
I really like this test. My only suggestion is that we clean up the artifacts that are created after this test is run. That said, I'm fine with leaving this as is and refactoring our unit tests in a future PR.